communication/onboard_parameters.cpp

```diff
@@ -284,9 +284,8 @@ static void onboard_parameters_receive_parameter(onboard_parameters_t* onboard_p
             if (match)
             {
                 // Only write if there is actually a difference
-                if (*(param->param) != set.param_value && set.param_type == param->data_type)
+                if (*(param->param) != set.param_value)
                 {
-                    // onboard_parameters_update_parameter(onboard_parameters, i, set.param_value);
                     (*param->param) = set.param_value;
```

Comment:
To answer: it is floored
If you set an integer param 1.1, it will be set to 1
If you set it to 1.9 it will be set to 1.